x-pack/plugins/stack_connectors/public/connector_types/cases_webhook/webhook_params.tsx

```diff
@@ -107,7 +107,7 @@ const WebhookParamsFields: React.FunctionComponent<ActionParamsProps<CasesWebhoo
       <EuiFormRow
         data-test-subj="title-row"
         fullWidth
-        error={errors['subActionParams.incident.title']}
+        error={errors['subActionParams.incident.title'] as string[]}
```

Comment:
I think this should be

```suggestion
        error={errors['subActionParams.incident.title'] as string}
```